ihatemoney/models.py

```diff
@@ -51,14 +51,25 @@
     ],
 )
 
+
 class BillType(Enum):
     EXPENSE = "Expense"
     REIMBURSEMENT = "Reimbursement"
 
+    def __str__(self):
+        return self.value
+
     @classmethod
     def choices(cls):
         return [(choice, choice.value) for choice in cls]
 
+    @classmethod
+    def coerce(cls, item):
+        try:
+            return item if isinstance(item, BillType) else BillType[item.upper()]
```

Comment:
Need `item.upper()` because the API expects that `bill_type` is passed as the enum value (i.e. "Expense" or "Reimbursement") but `item` is "EXPENSE" or "REIMBURSEMENT" when its passed as string to `coerce`.